contracts/adapters/UniswapAdapter.sol

```diff
@@ -10,64 +10,58 @@ import {IUniswapV2Pair} from "../interfaces/IUniswapV2Pair.sol";
 import {IUniswapV2Router02} from "../interfaces/IUniswapV2Router.sol";
 
 contract UniswapAdapter {
-
-    enum Exchange {Uniswap, Sushiswap}
-
     using SafeMath for uint256;
     using SafeERC20 for IERC20;
 
     address public immutable ethAddress;
     address public immutable diggAddress;
-    address public constant wethAddress = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+    address public constant wethAddress =
+        address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);

```

Comment:
Can you pass this in as a parameter in the constructor? We can't hardcode addresses because we want to deploy to kovan